test/integration/targets/win_scheduled_task/tasks/triggers.yml

```diff
@@ -288,6 +288,84 @@
     that:
     - trigger_monthlydow_again is not changed
 
+- name: create trigger repetition (check mode)
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition_check
+  check_mode: yes
+
+- name: get result of create trigger repetition (check mode)
+  win_scheduled_task_stat:
+    path: \
+    name: '{{test_scheduled_task_name}}'
+  register: trigger_repetition_result_check
+
+- name: assert results of create trigger repetition (check mode)
+  assert:
+    that:
+    - trigger_repetition_check is changed
+    - trigger_repetition_result_check.task_exists == False
```

Comment:
The reason why this would be failing is that the task actually exists from the `create monthly dow trigger` tasks above it that we didn't delete. The usual action that I've done instead of having another task to delete is to validate that the first trigger is still the monthly dow one, e.g.

```
- assert:
    that:
    - trigger_repetition_check is changed
    - trigger_repetition_check.task_exists == True
    - trigger_repetition_check.triggers|count == 1
    - trigger_repetition_check.triggers[0].type == "TASK_TRIGGER_MONTHLYDOW"
    - trigger_repetition_check.triggers[0].enabled == True
    - trigger_repetition_check.triggers[0].start_boundary == "2000-01-01T00:00:01"
    - trigger_repetition_check.triggers[0].end_boundary == None
    - trigger_repetition_check.triggers[0].weeks_of_month == "1,2"
    - trigger_repetition_check.triggers[0].days_of_week == "monday,wednesday"
```

This validates that the trigger wasn't actually changed in check mode but a change was reported.